challenges/1-produto-simples/java/azuosz/produtoSimples.java

```diff
@@ -0,0 +1,26 @@
+import  java.util.Scanner;
+
+public class produtoSimples {
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Digite um numero: ");
+        int  num1 = scanner.nextDouble();
```

Comment:
Erro de compilação: Type mismatch: cannot convert from double to int.
```suggestion
        int num1 = scanner.nextInt();
```